.circleci/config.yml

```diff
@@ -296,8 +299,9 @@ commands:
             --durations=0 \
             --master-host="<<parameters.master-host>>" \
             --master-port="<<parameters.master-port>>" \
+            -o junit_family=xunit1 \
             --junit-xml="<<parameters.junit-path>>" \
-            $(cat /tmp/tests-to-run)
+            $(< /tmp/tests-to-run)
```

Comment:
slight preference for switching back to `cat` as I feel like this syntax is pretty obscure